### PR TITLE
Restrict HPO search space of SB3 Methods (LR only)

### LINKED ISSUE #150: Search space for SB3 Methods is too large
Only really need the learning_rate for now, and the other values could be kept fixed at their default values (taken from SB3 docs)

### PR DESCRIPTION
**Pull request by lebrice**
Fixes #150 by dropping all keys of the search space dict, appart from
the learning rate.

Signed-off-by: Fabrice Normandin <fabrice.normandin@gmail.com>

### CODE REVIEW

**Comment by optimass** [comment]
it's more complicated than this.  each method have their own hparam to toggle with.
e.g. for PPO it's `n_steps` whereas for `DQN` it's the replay buffer size
(this is why I thought best to change the methods' file directly)

**Comment by lebrice** [comment]
As @optimass  mentionned, this is a bit too restrictive. We actually want to keep some hparams, apart from just the learning rate. 
Instead we'll comment out the fields with hparams we don't currently need, and set them to their default values instead.
